<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>868</width>
    <height>647</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MyEditor</string>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QWidget" name="fontSelectorPanel" native="true">
      <property name="hidden" stdset="0">
       <bool>true</bool>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QFontComboBox" name="fontFamilySelector">
         <property name="editable">
          <bool>false</bool>
         </property>
         <property name="frame">
          <bool>false</bool>
         </property>
         <property name="fontFilters">
          <set>QFontComboBox::MonospacedFonts</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="fontSizeSelector">
         <property name="editable">
          <bool>true</bool>
         </property>
         <property name="currentIndex">
          <number>5</number>
         </property>
         <property name="maxVisibleItems">
          <number>20</number>
         </property>
         <property name="frame">
          <bool>false</bool>
         </property>
         <item>
          <property name="text">
           <string>7</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>8</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>9</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>10</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>11</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>12</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>13</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>14</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>15</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>16</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>17</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>18</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="closeFontSelector">
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset theme="gtk-close">
           <normaloff>.</normaloff>.</iconset>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="fileName">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QsciScintilla" name="textEdit">
      <property name="toolTip">
       <string/>
      </property>
      <property name="whatsThis">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="searchPanel" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLineEdit" name="searchText">
         <property name="clearButtonEnabled">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="searchButton">
         <property name="text">
          <string>Buscar</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="closeSearchPanel">
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset theme="gtk-close">
           <normaloff>.</normaloff>.</iconset>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>868</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuArchivo">
    <property name="title">
     <string>Archivo</string>
    </property>
    <widget class="QMenu" name="menuNuevo">
     <property name="title">
      <string>Nuevo</string>
     </property>
     <addaction name="actionScript_Python"/>
     <addaction name="actionInterfaz_Grafica"/>
     <addaction name="actionCarpeta"/>
     <addaction name="actionProyecto"/>
    </widget>
    <addaction name="menuNuevo"/>
    <addaction name="actionAbrir"/>
    <addaction name="actionGuardar"/>
    <addaction name="actionGuardar_Como"/>
    <addaction name="separator"/>
    <addaction name="actionEjecutar"/>
    <addaction name="separator"/>
    <addaction name="actionSalir"/>
   </widget>
   <widget class="QMenu" name="menuEditar">
    <property name="title">
     <string>Editar</string>
    </property>
    <addaction name="actionFuente"/>
    <addaction name="separator"/>
    <addaction name="actionCortar"/>
    <addaction name="actionCopiar"/>
    <addaction name="actionPegar"/>
    <addaction name="actionSeleccionar_Todo"/>
    <addaction name="separator"/>
    <addaction name="actionBuscar"/>
    <addaction name="actionBuscarSiguiente"/>
   </widget>
   <widget class="QMenu" name="menuVer">
    <property name="title">
     <string>Ver</string>
    </property>
    <addaction name="actionArchivos"/>
    <addaction name="actionSalida"/>
    <addaction name="separator"/>
    <addaction name="actionZoom_in"/>
    <addaction name="actionZoom_out"/>
   </widget>
   <addaction name="menuArchivo"/>
   <addaction name="menuEditar"/>
   <addaction name="menuVer"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QDockWidget" name="panel">
   <property name="minimumSize">
    <size>
     <width>200</width>
     <height>259</height>
    </size>
   </property>
   <property name="features">
    <set>QDockWidget::DockWidgetMovable</set>
   </property>
   <property name="allowedAreas">
    <set>Qt::LeftDockWidgetArea|Qt::RightDockWidgetArea</set>
   </property>
   <property name="windowTitle">
    <string>Archivos</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2">
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <item>
      <widget class="QTreeView" name="treeView">
       <property name="sortingEnabled">
        <bool>true</bool>
       </property>
       <attribute name="headerShowSortIndicator" stdset="0">
        <bool>true</bool>
       </attribute>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="outputPanel">
   <property name="minimumSize">
    <size>
     <width>164</width>
     <height>232</height>
    </size>
   </property>
   <property name="features">
    <set>QDockWidget::DockWidgetClosable</set>
   </property>
   <property name="allowedAreas">
    <set>Qt::BottomDockWidgetArea</set>
   </property>
   <property name="windowTitle">
    <string>Salida</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QTextBrowser" name="outputText">
       <property name="lineWrapMode">
        <enum>QTextEdit::NoWrap</enum>
       </property>
       <property name="openExternalLinks">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="clearOutput">
       <property name="text">
        <string>Clear</string>
       </property>
       <property name="icon">
        <iconset theme="gtk-clear">
         <normaloff>.</normaloff>.</iconset>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextBesideIcon</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="allowedAreas">
    <set>Qt::TopToolBarArea</set>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonFollowStyle</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionAbrir"/>
   <addaction name="actionGuardar"/>
   <addaction name="actionEjecutar"/>
  </widget>
  <widget class="QDockWidget" name="imagePanel">
   <property name="allowedAreas">
    <set>Qt::RightDockWidgetArea</set>
   </property>
   <property name="windowTitle">
    <string>Imágen</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_3">
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QLabel" name="imageLabel">
       <property name="frameShape">
        <enum>QFrame::WinPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="scaledContents">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionAbrir">
   <property name="icon">
    <iconset theme="gtk-open">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Abrir...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionGuardar">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset theme="gtk-save">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Guardar</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSalir">
   <property name="icon">
    <iconset theme="application-exit">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Salir</string>
   </property>
  </action>
  <action name="actionFuente">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset theme="fonts">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Fuente</string>
   </property>
  </action>
  <action name="actionEjecutar">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset theme="gtk-execute">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Ejecutar</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </action>
  <action name="actionGuardar_Como">
   <property name="icon">
    <iconset theme="gtk-save">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Guardar Como..</string>
   </property>
  </action>
  <action name="actionArchivos">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset theme="folder">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Archivos</string>
   </property>
  </action>
  <action name="actionCortar">
   <property name="icon">
    <iconset theme="gtk-cut">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Cortar</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionCopiar">
   <property name="icon">
    <iconset theme="gtk-copy">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Copiar</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionPegar">
   <property name="icon">
    <iconset theme="gtk-paste">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Pegar</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="actionSeleccionar_Todo">
   <property name="icon">
    <iconset theme="gtk-select-all">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Seleccionar Todo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="actionZoom_in">
   <property name="icon">
    <iconset theme="zoom-in">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Zoom +</string>
   </property>
   <property name="shortcut">
    <string>Ctrl++</string>
   </property>
  </action>
  <action name="actionZoom_out">
   <property name="icon">
    <iconset theme="zoom-out">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Zoom -</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+-</string>
   </property>
  </action>
  <action name="actionScript_Python">
   <property name="icon">
    <iconset theme="document-new">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Script de Python</string>
   </property>
  </action>
  <action name="actionInterfaz_Grafica">
   <property name="icon">
    <iconset theme="window-new">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Interfaz Gráfica</string>
   </property>
  </action>
  <action name="actionCarpeta">
   <property name="icon">
    <iconset theme="folder-new">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Carpeta</string>
   </property>
  </action>
  <action name="actionProyecto">
   <property name="icon">
    <iconset theme="address-book-new">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Proyecto</string>
   </property>
  </action>
  <action name="actionBuscar">
   <property name="icon">
    <iconset theme="gtk-find">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Buscar</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="actionBuscarSiguiente">
   <property name="icon">
    <iconset theme="gtk-find">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Buscar Siguiente</string>
   </property>
   <property name="shortcut">
    <string>F3</string>
   </property>
  </action>
  <action name="actionSalida">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset theme="utilities-terminal">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Salida</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QsciScintilla</class>
   <extends>QFrame</extends>
   <header>Qsci/qsciscintilla.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>actionSalir</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionFuente</sender>
   <signal>toggled(bool)</signal>
   <receiver>fontSelectorPanel</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>439</x>
     <y>53</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>closeFontSelector</sender>
   <signal>clicked()</signal>
   <receiver>actionFuente</receiver>
   <slot>toggle()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>769</x>
     <y>53</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionArchivos</sender>
   <signal>toggled(bool)</signal>
   <receiver>panel</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>106</x>
     <y>323</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionCopiar</sender>
   <signal>triggered()</signal>
   <receiver>textEdit</receiver>
   <slot>copy()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>573</x>
     <y>348</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionPegar</sender>
   <signal>triggered()</signal>
   <receiver>textEdit</receiver>
   <slot>paste()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>573</x>
     <y>348</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionSeleccionar_Todo</sender>
   <signal>triggered()</signal>
   <receiver>textEdit</receiver>
   <slot>selectAll()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>573</x>
     <y>348</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionCortar</sender>
   <signal>triggered()</signal>
   <receiver>textEdit</receiver>
   <slot>cut()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>573</x>
     <y>348</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionZoom_in</sender>
   <signal>triggered()</signal>
   <receiver>textEdit</receiver>
   <slot>zoomIn()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>573</x>
     <y>348</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionZoom_out</sender>
   <signal>triggered()</signal>
   <receiver>textEdit</receiver>
   <slot>zoomOut()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>573</x>
     <y>348</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>closeSearchPanel</sender>
   <signal>clicked()</signal>
   <receiver>searchPanel</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>619</x>
     <y>591</y>
    </hint>
    <hint type="destinationlabel">
     <x>573</x>
     <y>592</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>searchButton</sender>
   <signal>clicked()</signal>
   <receiver>actionBuscarSiguiente</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>559</x>
     <y>592</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>searchText</sender>
   <signal>returnPressed()</signal>
   <receiver>actionBuscarSiguiente</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>404</x>
     <y>592</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionSalida</sender>
   <signal>toggled(bool)</signal>
   <receiver>outputPanel</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>433</x>
     <y>507</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>clearOutput</sender>
   <signal>clicked()</signal>
   <receiver>outputText</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>41</x>
     <y>602</y>
    </hint>
    <hint type="destinationlabel">
     <x>433</x>
     <y>486</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
